test/e2e/antreapolicy_test.go

```diff
@@ -142,9 +145,23 @@ func initialize(t *testing.T, data *TestData, customNamespaces map[string]TestNa
 	// k8sUtils is a global var
 	k8sUtils, err = NewKubernetesUtils(data)
 	failOnError(err, t)
-	ips, err := k8sUtils.Bootstrap(namespaces, podsPerNamespace, true, nil, nil)
+	ips, realizedName, err := k8sUtils.Bootstrap(namespaces, podsPerNamespace, true, nil, nil)
```

Comment:
This change is completely unnecessary. We could simply use `k8sutils.GetPodByLabel`

Reply:
Updated it works, thanks.